doc/src/install_guide/installation_guide.md

```diff
@@ -117,30 +105,43 @@ Before you build the OPAE SDK, you must install the required packages. Run the f
 # pip3 install jsonschema virtualenv pyyaml pybind11
 ```
 
+### RHEL 8.2 ###
+Register and enable Red Hat subscription to install any packages on the system.
+
+```console
+# subscription-manager register --proxy=PROXY --username=USER --password=PASSWORD --auto-attach
+```
+
+Set the RHEL version and install packages
+
+```console
+# subscription-manager release --set=8.2 --proxy proxy-xyz.com:xyz
```

Comment:
This line needs some explanation what xyz should be:
This should be the builder's proxy domain name and port number something like this.

Reply:
fixed